tools/codegen/api/translate.py

```diff
@@ -94,6 +94,12 @@ def translate(
             ctx[NamedCType(t.elem.elem.name, ConstRefCType(BaseCType(tensorT)))] = \
                 f'({b.expr}.has_value() ? *{b.expr} : at::Tensor())'
```

Comment:
Oh interesting. Actually, the `isinstance(t, NamedCType)` check below is not necessary, because the code above is dead. You can tell this because Binding's type is NamedCType.